spec/features/trainee_actions/withdraw_trainee_spec.rb

```diff
@@ -321,4 +357,25 @@ def and_i_choose_they_have_not_started
   def then_i_am_taken_to_the_forbidden_withdrawal_page
     expect(withdrawal_forbidden_page).to be_displayed
   end
+
+  def and_i_click_change_start_date
+    withdrawal_confirmation_page.start_date_change_link.click
+  end
+
+  def then_i_am_redirected_to_start_date_verification_page
+    expect(start_date_verification_page).to be_displayed(id: trainee.slug)
+  end
+
+  def and_i_select_the_trainee_started_their_itt
```

Comment:
You can remove this function and use `and_i_choose_they_have_started` instead - looks like it's identical 👍